api/src/utils/index.ts

```diff
@@ -111,11 +111,18 @@ function keyMapping(key: string, value: string, objValue?: string): any {
  */
 
 /**
- * Get index from [a, b, c, ..., z]
+ * Get index from [a, b, c, ..., z, aa, ab, ..., az, ba, bb, ..., bz...]
+ * actually, this is base 26
  * @param char
  * @returns 1, 2, ..., 26
  */
-export const getIndexOfAlphabet = (char: string) => char.toLowerCase().charCodeAt(0) - 96;
+export const getIndexOfAlphabet = (char: string): number => {
```

Comment:
This definitely needs some unit tests

Reply:
yep, added.
![image](https://user-images.githubusercontent.com/46575401/114216937-69e9dc00-991c-11eb-952c-3840d5afa47a.png)